string name
string value
string from_node
float64 timestamp